src/external.d.ts

```diff
@@ -0,0 +1,23 @@
+declare module "dryadic" {
```

Comment:
dryadic will also be ported to Typescript. Afterwards this declaration file will be removed.